### PR TITLE
(#2379) add storage_medium info  when getting table info

### PR DESCRIPTION
**Pull request by wangbo**
see #2379

### CODE REVIEW

File: fe/src/main/java/org/apache/doris/catalog/Catalog.java [hunk 1]
```diff
@@ -4001,6 +4007,7 @@ public static void getDdlStmt(Table table, List<String> createTableStmt, List<St
                 sb.append("(\"version_info\" = \"");
                 sb.append(Joiner.on(",").join(partition.getVisibleVersion(), partition.getVisibleVersionHash()))
                         .append("\"");
+                sb.append(",\"storage_medium\" = \"").append(partitionInfo.getDataProperty(partition.getId()).getStorageMedium()).append("\"");
```
Comment (imay) [inline fe/src/main/java/org/apache/doris/catalog/Catalog.java]: use PROPERTIES_STORAGE_MEDIUM?

File: fe/src/main/java/org/apache/doris/catalog/RangePartitionInfo.java [hunk 2]
```diff
@@ -386,6 +386,7 @@ public String toSql(OlapTable table, List<Long> partitionId) {
             sb.append(range.lowerEndpoint().toSql());
             sb.append(", ").append(range.upperEndpoint().toSql()).append(")");
 
+            sb.append(", storage_medium=").append(idToDataProperty.get(partition.getId()).getStorageMedium());
```
Comment (imay) [inline fe/src/main/java/org/apache/doris/catalog/RangePartitionInfo.java]: PROPERTIES_STORAGE_MEDIUM?

File: fe/src/main/java/org/apache/doris/catalog/Catalog.java (hunk 1 above)
Comment (wangbo) [reply fe/src/main/java/org/apache/doris/catalog/Catalog.java]: 👌;reduce append calls times seems pointless here

File: fe/src/main/java/org/apache/doris/catalog/RangePartitionInfo.java (hunk 2 above)
Comment (wangbo) [reply fe/src/main/java/org/apache/doris/catalog/RangePartitionInfo.java]: 👌

File: fe/src/main/java/org/apache/doris/catalog/RangePartitionInfo.java [hunk 3]
```diff
@@ -380,12 +383,16 @@ public String toSql(OlapTable table, List<Long> partitionId) {
...
             Range<PartitionKey> range = entry.getValue();
+            DataProperty paritionDataProperty = idToDataProperty.get(partition.getId());
 
             // print all partitions' range is fixed range, even if some of them is created by less than range
             sb.append("PARTITION ").append(partitionName).append(" VALUES [");
             sb.append(range.lowerEndpoint().toSql());
             sb.append(", ").append(range.upperEndpoint().toSql()).append(")");
 
+
+            sb.append(", ").append(PROPERTIES_STORAGE_MEDIUM).append("=").append(paritionDataProperty.getStorageMedium());
```
Comment (imay) [inline fe/src/main/java/org/apache/doris/catalog/RangePartitionInfo.java]:
```suggestion
            sb.append(", \"").append(PROPERTIES_STORAGE_MEDIUM).append("\"=").append(paritionDataProperty.getStorageMedium());
```

File: fe/src/main/java/org/apache/doris/catalog/RangePartitionInfo.java [hunk 4]
```diff
@@ -380,12 +383,16 @@ public String toSql(OlapTable table, List<Long> partitionId) {
...
+            sb.append(", ").append(PROPERTIES_STORAGE_COLDOWN_TIME).append("=").append(paritionDataProperty.getCooldownTimeString());
```
Comment (imay) [inline fe/src/main/java/org/apache/doris/catalog/RangePartitionInfo.java]:
```suggestion
            sb.append(", \"").append(PROPERTIES_STORAGE_COLDOWN_TIME).append("\"=").append(paritionDataProperty.getCooldownTimeString());
```

**Comment by kangkaisen** [comment]
Please add the UT.

**Comment by wangbo** [comment]
1 add ut
2 fix storage_coldtime error bug,using getTimeMillis instead of getLoneValue

**Review by morningman — CHANGES_REQUESTED** [review]
You can use comand:
`SHOW PARTITIONS FROM tbl;`
to get the storage medium info, why using `CREATE TABLE stmt`?

**Comment by wangbo** [comment]
> You can use comand:
> `SHOW PARTITIONS FROM tbl;`
> to get the storage medium info, why using `CREATE TABLE stmt`?

I Didn't know this before
I think show parition stmt is enough

**Pull request closed without merging**